### ISSUE TITLE
GOG achievements don't import

### ISSUE THREAD

**Issue by Sprabuni** [Lacro59/playnite-successstory-plugin#556]
**Describe the bug**
Achievements from GOG don't get imported even though I'm authenticated

**To Reproduce**
- Manually refresh data on a GOG game

**Expected behavior**
I expect achievements from GOG to get imported to Playnite, both uncompleted and completed ones.

**Screenshots**
![image](https://github.com/user-attachments/assets/90f21bd6-d0cf-4422-a62d-61907b7db96f)

**Extensions log**
[extensions.log](https://github.com/user-attachments/files/17906849/extensions.log)

**Comment by StraussVanDor** [comment]
Same here.
In my case, the "Get account ID" button takes me to the Epic Games account instead of GOG.

**Comment by Vinsup** [comment]
> Same here. In my case, the "Get account ID" button takes me to the Epic Games account instead of GOG.

Same on my side

**Comment by RetroFlock** [comment]
The same thing is happening to me. It won't import any achievements for GOG games.

**Comment by Lacro59** [comment]
Can you try this version?
[playnite-successstory-plugin_3_4.zip](https://github.com/user-attachments/files/18195717/playnite-successstory-plugin_3_4.zip)

**Comment by Vinsup** [comment]
Still 404 error.
I was not using GalaxyUserId previously but after getting it by clicking on the link in the plugin settings, saving settings and relaunching an update of the plugin it still prompt you with a 404 error.

**Comment by RetroFlock** [comment]
I don't know about you, but it worked for me.

**Comment by Vinsup** [comment]
OK. Indeed. It works.

I had to check the "Is Private" checkbox. It did not worked at first.
I did relaunch Playnite a second time (first time was after updating the plugin with the provided file), then login to GOG to update games from GOG (it got disconnected for an unknown reason), then I clicked to update missing games in SuccessStory plugin and it dit prompt me again to login to GOG from the SuccessStory settings but finally I got my missing success for my GOG games.
